src/common/options.cc

```diff
@@ -1019,6 +1019,11 @@ std::vector<Option> get_global_options() {
     .set_default(100_M)
     .set_description("Limit messages that are read off the network but still being processed"),
 
+    Option("ms_dispatch_throttle_log_interval", Option::TYPE_SECS, Option::LEVEL_ADVANCED)
+    .set_default(30)
+    .set_min(0)
```

Comment:
Does `0` disable?

Reply:
No, it doesn't disable. However, using a value `0` for `ms_dispatch_throttle_log_interval` doesn't make sense too. (looking at `ProtocolV2::throttle_dispatch_queue()`). I couldn't recollect, but I think I set a min value at an early stage just to make it positive values. As we are using `'ms_dispatch_throttle_log_interval', 5` from the test function `test_dispatch_queue_throttle_message`, set the min value to 5 or just drop the set_min() ?

Reply:
@batrick ping

Reply:
The implication of my question is that perhaps 0 should disable the log message?

Reply:
Updated